src/project.ts

```diff
@@ -67,6 +67,7 @@ export class Project extends MobileProject {
       androidIconForeground: await this.loadInputAsset('android/icon-foreground', AssetKind.Icon, Platform.Android),
       androidIconBackground: await this.loadInputAsset('android/icon-background', AssetKind.Icon, Platform.Android),
 
+      androidBanner: await this.loadInputAsset('android/banner', AssetKind.Banner, Platform.Android),
```

Comment:
The README update shows that the `banner.png` file should be placed in the `assets/banner.png` folder directly. 

But it is not read that way. One has to place it in the `assets/android/banner.png` folder.

Is this intended? Then it should be noted in the README docs. Alternatively, and I think more logically, it should work in both folders regardless where it is placed.

Reply:
Also found that the banner addition seems to add the manifest line regardless `android:banner="@drawable/banner"` regardless if it's actually adding the banner (aka if the file is not in `assets/android/banner.png` it will still add the line in the `AndroidManifest.xml`)